### DISCUSSION TITLE
Grove Shield for XIAO with battery management chip | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1196] [Q&A]
# Grove Shield for XIAO with battery management chip | Seeed Studio Wiki

Grove Shield for Seeed Studio XIAO with embedded battery management chip

https://wiki.seeedstudio.com/Grove-Shield-for-Seeeduino-XIAO-embedded-battery-management-chip/

**Comment by scottw-finao** [comment]
Does this have any monitoring of the battery level that is visible to the xiao, such as by one of the analog pins?
